aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -741,69 +711,33 @@ The following example permits HTTP/1.1 and HTTP/2 connections on port 8000. Conn
 });
 ```
 
-Optionally use Connection Middleware to filter TLS handshakes on a per-connection basis for specific ciphers:
+Use `CipherSuitesPolicy` to filter TLS handshakes on a per-connection basis for specific ciphers if required:
```

Comment:
FYI this only works on Linux.

Reply:
You said, "replaced by" ... so that's what I did.

Reply:
Yeah, my bad.